schema/plugin.json

```
@@ -1,6 +1,6 @@
 {
-  "title": "Launcher",
-  "description": "Launcher settings.",
+  "title": "@jlab-enhanced/launcher",
+  "description": "@jlab-enhanced/launcher settings.",
```

This does not need to be updated:

```suggestion
  "title": "Launcher",
  "description": "Launcher settings.",
```